### PR TITLE
Netigen Api Integration

### PR DESCRIPTION
**Pull request by WrobelKrakow**
Using old style, we need new api_androdi with hilt for better usage

### CODE REVIEW

File: dependencies.gradle [hunk 1]
```diff
@@ -91,4 +91,13 @@ ext {
...
+
+    //api_android
+    def hmsPluginVersion = "1.4.1.300"
+    def apiAndroidVersion = "3.4.3"
+
+    hmsPlugin = "com.huawei.agconnect:agcp:" + hmsPluginVersion
+    coreApiAndroid = "com.github.netigenkluzowicz.api_android:core" + apiAndroidVersion
+    gmsApiAndroid = "com.github.netigenkluzowicz.api_android:gms" + apiAndroidVersion
+    hmsApiAndroid = "com.github.netigenkluzowicz.api_android:hms " + apiAndroidVersion
 }
```
Comment (michal-bogucki) [inline dependencies.gradle]: This can be passed as a list.

**Review by michal-bogucki — CHANGES_REQUESTED** [review]

File: app/src/main/java/pl/netigen/sampleapp/core/app/App.kt [hunk 2]
```diff
@@ -9,8 +9,20 @@ import timber.log.Timber
...
+            Timber.plant(object : Timber.DebugTree() {
+                override fun createStackElementTag(element: StackTraceElement): String {
+                    return "$TAG.${element.className.substringAfterLast(".")}.${element.methodName}"
+                }
+            })
         }
     }
+
+    companion object {
+        const val TAG = "sampleApp"
```
Comment (michal-bogucki) [inline app/src/main/java/pl/netigen/sampleapp/core/app/App.kt]: Maybe let's assume that the tag will be package name

**Review by michal-bogucki — CHANGES_REQUESTED** [review]

File: dependencies.gradle (hunk 1 above)
Comment (WrobelKrakow) [reply dependencies.gradle]: It is flavor depended compilation

File: app/src/main/java/pl/netigen/sampleapp/core/app/App.kt (hunk 2 above)
Comment (WrobelKrakow) [reply app/src/main/java/pl/netigen/sampleapp/core/app/App.kt]: It's very useful when You want filter only Timber logs, with package name i have a stream of logs form AudioMAnager f.e.

**Pull request merged**